src/python-common/ceph/deployment/inventory.py

```diff
@@ -52,6 +53,7 @@ def __init__(self,
                  lvs=None,  # type: Optional[List[str]]
                  device_id=None,  # type: Optional[str]
                  lsm_data=None,  # type: Optional[Dict[str, Dict[str, str]]]
+                 created=None # type Optional[datetime.datetime]
```

Comment:
```suggestion
                 created=None  # type: Optional[datetime.datetime]
```
To make mypy and flake8 happy. You might also have to import datetime to get the type signature to work